client/src/boot/registerComponents.js

```diff
@@ -98,5 +100,7 @@ export default () => {
     HtmlEditorField,
     NumberField,
     PopoverOptionSet,
+    ToastsContainer,
+    FileStatusIcon
```

Comment:
`FileStatusIcon` was implemented in another PR, but it didn't get registered in the Injector.

Reply:
That bit might not be necessary. I've added it in there because I was trying to debugging an out-of-sync `@silverstripe/webpack-config` dependency. I'll try removing it and see if it makes any difference.

Reply:
Doesn't look like it makes a difference. Removing.